src/Shared/Microsoft.AspNetCore.Razor.Serialization.Json/SerializationFormat.cs

```diff
@@ -9,5 +9,5 @@ internal static class SerializationFormat
     // or any of the types that compose it changes. This includes: RazorConfiguration,
     // ProjectWorkspaceState, TagHelperDescriptor, and DocumentSnapshotHandle.
     // NOTE: If this version is changed, a coordinated insertion is required between Roslyn and Razor for the C# extension.
-    public const int Version = 4;
+    public const int Version = 5;
```

Comment:
Nit: Could revert to `3` 😛

It might actually be easier to do a `git revert` of your previous commit here, because i don't think anyone else will have touched it, and it might avoid you having to update the json resource files.

Reply:
Honestly I'm good with reverting this or keeping as is. I just want to make whatever the final result is consistently work. We'll make this a triad vote: 

@DustinCampbell  to revert or not revert? That is the question

Reply:
I'd say to revert it. This is effectively now just the same as version 3. Note that the JSON serialization format version number is never actually used in product code, just test code. The JSON serialization infrastructure is only used in the product for Live Share, and that doesn't serialize `RazorProjectInfo`, which is where that version number matters.